lib/transformations/extractTextPlugin/extractTextPlugin.js

```diff
@@ -1,42 +1,36 @@
-"use strict";
-
+// @flow
+import type { Ijscodeshit, IPath, INewExpression, IObjectExpression, IMemberExpression, ILiteral } from "../../types"; // eslint-disable-line node/no-unsupported-features
```

Comment:
Split these long imports up... 
```js
 // eslint-disable-next-line node/no-unsupported-features
import type { Ijscodeshit, 
              IPath, 
              INewExpression, 
              IObjectExpression, 
              IMemberExpression, 
              ILiteral 
             } from "../../types";
```

Reply:
In order to disable ESLint for a single of code, you have to put the comment on the same line. Putting the comment on a different line won't work. Could we agree on something similar? 

```js
import type { 
  Ijscodeshit,
  IPath,
  INewExpression, 
  IObjectExpression, 
  IMemberExpression, 
  ILiteral 
} from "../../types";  // eslint-disable-next-line node/no-unsupported-features
```